backend/vehicles/src/power-units/dto/response/read-power-unit.dto.ts

```diff
@@ -0,0 +1,80 @@
+import { AutoMap } from '@automapper/classes';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class ReadPowerUnitDto {
+  @AutoMap()
+  @ApiProperty({
+    example: '1',
+    description: 'The Power Unit ID',
+  })
+  powerUnitId: string;
+
+  @AutoMap()
+  @ApiProperty({
+    description: 'The Unit Number',
+    example: 'KEN1',
```

Comment:
I think a better description would make more sense. The field is called unit number but the example suggests alphanumeric value.

Reply:
Resolved